CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.0] - 2021-11-17
```

Comment:
What happened to the changelog?

Reply:
The release `0.4.0` is created with what was inside the Unrelease section.

Reply:
Oh, no problem here ^^